boards/beagle/pocketbeagle_2/doc/index.rst

```diff
@@ -3,21 +3,25 @@
 Overview
 ********
 
-PocketBeagle 2 is a computational platform powered by TI AM62x SoC (there are two
-revisions, AM6232 and AM6254).
-
-The board configuration provides support for the ARM Cortex-M4F MCU core.
+PocketBeagle 2 is a computational platform powered by TI AM6254 SoC (rev A0,
+which has been replaced by rev A1 used AM6232).
```

Comment:
so why are you not defining revision A1 (and I guess make it the default?) in board.yml - did I miss something?

Reply:
Well, I only have rev A0 with me. Theoretically, rev A1 should not require any changes, but well, since I cannot test it myself, I am not sure about adding it.

Reply:
Oh my bad, sorry, I thought this PR was effectively introducing support for rev A1 / AM6254, so I clearly read this too quickly :/ That being said, the comment still kind of stands - this wording change (and the update to the section hardware is even worse) doesn't really belong to this commit and is confusing since it's talking about a version that's effectively not supported in Zephyr..?

Reply:
I have rolled back the unrelated changes for now. Will make those in rev A1 PR.